packages/cadl-typescript/test/integration/fixedEnums.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { assert } from "chai";
+import EnumsFixedClientFactory, {
+  EnumsFixedClient
+} from "./generated/enums/fixed/src/index.js";
+describe("FixedEnums Rest Client", () => {
+  let client: EnumsFixedClient;
+
+  beforeEach(() => {
+    client = EnumsFixedClientFactory({ allowInsecureConnection: true });
+  });
+
+  it("should get known value", async () => {
+    try {
+      const result = await client.path("/enums/fixed/string/known-value").get();
+      assert.strictEqual(result.status, "200");
+      assert.strictEqual(result.body, "Monday");
+    } catch (err) {
+      assert.fail(err as string);
+    }
+  });
+
+  // FIXME core-client can't JSON.stringfy this value
+  it.skip("should put known value", async () => {
```

Comment:
Skip these two cases because we have issue: https://github.com/Azure/autorest.typescript/issues/1712